modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
++
+To optionally configure the Operator's deployment with a config file, make changes to the `config/default/kustomization.yaml` file as shown in the following diff:
++
+.`config/default/kustomization.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+# If you want your controller-manager to expose the /metrics # endpoint w/o any authn/z, please comment the following line.
+\- manager_auth_proxy_patch.yaml
++# Mount the controller config file for loading manager configurations
++# through a ComponentConfig type
++- manager_config_patch.yaml
+----
+====
++
+Flags can be used as is or to override config file values.
+
+. Make the following changes to your `config/rbac/leader_election_role.yaml` file:
+
+.. For Ansible- and Helm-based Operator projects, add role rules for leader election.
++
+Add the rule for the `apiGroups`, `coordination.k8s.io`, and resource `leases` in the `config/rbac/leader_election_role.yaml` file:
++
+.`config/rbac/leader_election_role.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+rules:
+  - apiGroups:
+      - ""
+      - coordination.k8s.io
+    resources:
+      - configmaps
+      - leases
+----
+====
+
+.. Move `leases.coordination.k8s.io` to its own `proxy-role` rule.
++
+Make the following changes to `config/rbac/leader_election_role.yaml` file:
++
+.`config/rbac/leader_election_role.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+rules:
+\- apiGroups:
+  - ""
+-  - coordination.k8s.io
+  resources:
+  - configmaps
++ verbs:
++- get
++- list
++- watch
++- create
++- update
++- patch
++- delete
++- apiGroups:
++- coordination.k8s.io
++ resources:
+    - leases
+  verbs:
+    - get
+----
+====
+
+. For Ansible-based Operator projects, update Ansible collections.
++
+In your `requirements.yml` file, change the `version` field for `community.kubernetes` to `1.2.1`, and the `version` field for `operator_sdk.util` to `0.2.0`.
+
+. Make the following changes to your `config/default/manager_auth_proxy_patch.yaml` file:
+
+** For Ansible-based Operator projects, add the `--health-probe-bind-address=:6789` argument to the `config/default/manager_auth_proxy_patch.yaml` file:
++
+.`config/default/manager_auth_proxy_patch.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        args:
+        - "--health-probe-bind-address=:6789"
+        ...
+----
+====
+
+** For Helm-based Operator projects:
+
+... Add the `--health-probe-bind-address=:8081` argument to the `config/default/manager_auth_proxy_patch.yaml` file:
++
+.`config/default/manager_auth_proxy_patch.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        args:
+        - "--health-probe-bind-address=:8081"
+        ...
+----
+====
+
+... Replace deprecated leader election and metrics address flags.
```

Comment:
I can't quite spot why you ended this instructional text with a period and the next (line 831) with a colon.

I'm not asserting that it's a problem, I just haven't discerned the distinction.

Reply:
You are observing a mid-conversion-from-source-material that didn't quite get fully baked like the others. :sweat: Will join as a single paragraph.